### PR TITLE
Bug fix in split_node_rq

### PR DESCRIPTION
**Pull request by mganahl**
`split_node_rq` is currently doing the wrong thing; instead of returning `r, q` it returns `q, r`.

### CODE REVIEW

File: tensornetwork/tensornetwork_test.py [hunk 1]
```diff
@@ -1508,6 +1509,98 @@ def test_split_node_rq(backend):
   np.testing.assert_allclose(a.tensor, net.contract(left[3]).tensor)
 
 
+def test_split_node_qr_unitarity_complex_numpy():
```
Comment (Thenerdstation) [inline tensornetwork/tensornetwork_test.py]: Please you the `backend` argument to reduce code copying.

File: tensornetwork/network.py [hunk 2]
```diff
@@ -1206,8 +1206,8 @@ def split_node_full_svd(
     return left_node, singular_values_node, right_node, trun_vals
 
   def remove_node(self, node: network_components.BaseNode
-                 ) -> Tuple[Dict[Text, network_components.
-                                 Edge], Dict[int, network_components.Edge]]:
+                 ) -> Tuple[Dict[Text, network_components
```
Comment (Thenerdstation) [inline tensornetwork/network.py]: Please fix.

File: tensornetwork/tensornetwork_test.py (hunk 1 above)
Comment (mganahl) [reply tensornetwork/tensornetwork_test.py]: can do as soon as we have backend.conj() (see #213)

File: tensornetwork/network.py (hunk 2 above)
Comment (mganahl) [reply tensornetwork/network.py]: Sorry about these formatting issues. I'm using py-yapf on emacs which makes these weird formatting decisions sometimes. I haven't figured out how to change it yet, and manual changes are a pain. Can we postpone this for a little longer? Rather have the bug fixed as soon as possible

**Comment by mganahl** [comment]
If the travie build goes through, that would be it so far from my side

File: tensornetwork/tensornetwork_test.py [hunk 3]
```diff
@@ -1518,6 +1518,141 @@ def test_split_node_rq(backend):
...
+  n1 = net_2.add_node(q.tensor)
+  n2 = net_2.add_node(q.tensor)
+  n2[0] ^ n1[0]
+  u2 = net_2.contract_between(n1, n2)
+
+  np.testing.assert_almost_equal(u1.tensor, np.eye(3))
+  np.testing.assert_almost_equal(u2.tensor, np.eye(3))
+
+
+# def test_split_node_qr_unitarity_float_numpy():
```
Comment (Thenerdstation) [inline tensornetwork/tensornetwork_test.py]: Delete the commented out code.

**Comment by Thenerdstation** [comment]
I'll let https://github.com/google/TensorNetwork/pull/222 in first. Youll likely need to move the tests to our new `tests/` directory.

**Comment by mganahl** [comment]
are the python3.5 tests now deprecated?

**Comment by Thenerdstation** [comment]
Yes, atleast until the JAX team fixes it on their end.

**Comment by Thenerdstation** [comment]
#222 is in now, fix the conflicts and we should be good to go.

**Pull request merged**